src/mail/bin/html-to-plain-text

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+set -eo pipefail
+# Run html-to-text to convert all html files to text files
+DIR_MAILS="../backend/marsha/core/templates/core/mail/"
+
+if [ ! -d "${DIR_MAILS}" ]; then
+  mkdir -p "${DIR_MAILS}";
+fi
+
+if [ ! -d "${DIR_MAILS}"html/ ]; then
+  mkdir -p "${DIR_MAILS}"html/;
+fi
```

Comment:
If both directories don't exist then there is no mail to convert in text format. You can exit the script in that case